app/routes/app+/showcase.tsx

```diff
@@ -228,30 +226,30 @@ function SubmissionsCard({ submissions, projects }: { submissions: CombinedDoc[]
   return (
     <div className="space-y-4">
       {submissions.map((s) => {
+        const averageScore = (s.score?.reviewSum || 0) / (s.score?.reviewCount || 0);
```

Comment:
divide by 0 issue?

Reply:
oops yup, fixed this in most places but must have missed this one, thank you